scripts/setup_cyclecloud.sh

```diff
@@ -72,7 +80,12 @@ restorecon -r -v ${CS_HOME}
 
 # explicitly install java 8 before cyclecloud (to ensure environment is set)
 yum -y install java-1.8.0-openjdk-headless
-yum -y install cyclecloud8-${CC_MARKETPLACE_VERSION}
+
+if [[ $REPO_STREAM == "local" ]]; then
```

Comment:
condition on "CC_PACKAGE_NAME" is not empty instead (it's not really a repo stream :) )